packages/git/src/browser/git-repository-provider.ts

```diff
@@ -30,6 +31,9 @@ export class GitRepositoryProvider {
     protected _allRepositories: Repository[] = [];
     protected readonly onDidChangeRepositoryEmitter = new Emitter<Repository | undefined>();
 
+    protected root: FileStat | undefined;
```

Comment:
please simplify:
- it can be accessed on each call of refresh method as before
- using `root` seems to be redundant